assets/src/dashboard/app/views/myStories/content/storyGridItem/index.js

```diff
@@ -61,6 +55,8 @@ const StoryGridItem = ({
   story,
   storyMenu,
 }) => {
+  const { enablePostLocking } = useFeatures();
+  const { userId } = useConfig();
```

Comment:
moved necessary checks for if a story is locked up a level so we can disable the context menu item to rename a story. this will need to be added to the list view as well. Will do that work here: https://github.com/google/web-stories-wp/issues/8423